code/clue.js

```diff
@@ -1,59 +1,357 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+let killerIsPicked = false;
+let weaponIsPicked = false;
+let roomIsPicked = false;
 
-const mrGreen = {
+//Suspect objects
+
+let mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
-  color: 'green',
+  color: '#0b6623',
   description: 'He has a lot of connections',
-  age: 45,
+  age: 48,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  occupation: 'Entrepreneur',
+  favouriteWeapon: 'rope'
+};
+
+let professorPlum = {
```

Comment:
You don't overwrite those values anywhere in app, they should be declared with `const`